cli/src/commands/scan.rs

```diff
@@ -290,8 +322,14 @@ pub fn exec_scan(args: &ArgMatches) -> anyhow::Result<()> {
                 .unwrap()
                 .push((file_path.clone(), now));
 
+            let target_file = file_path.as_path();
```

Comment:
This line can be removed.

Reply:
fixed in [cca5adf](https://github.com/VirusTotal/yara-x/pull/183/commits/cca5adfaeb3671482a47f5b0a1a8e3f7d2a70fe8)